### PR TITLE
documentation: update changes and release notes

### PR DESCRIPTION
**Pull request by iGormilhit**
* NEW Adds v0.5.1 changes into CHANGES.rst.
* NEW Upadates RELEASE-NOTES.rst with v0.5.1 changes.

Co-Authored-by: Igor Milhit <igor.milhit@rero.ch>

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?

### CODE REVIEW

File: RELEASE-NOTES.rst [hunk 1]
```diff
@@ -18,6 +18,55 @@
...
 
+v0.5.1
+------
+
+-  User interface:
+
+   -  Adds a new method to change session locale, in preparation for the
+      future pure JS application.
+   -  Rewrites frontpage to emphasize the public demo site and improves
+      it rendering on small screens.
```
Comment (mmo) [inline RELEASE-NOTES.rst]: its rendering on small screens

File: RELEASE-NOTES.rst [hunk 2]
```diff
@@ -18,6 +18,55 @@
...
+      instance and version of RERO ILS on which the issue occurred.
+
+-  Fixed issues:
+
+   -  `#488 <https://github.com/rero/rero-ils/issues/488>`__: disallows
+      editing libraries by a non affiliated librarian.
+   -  `#475 <https://github.com/rero/rero-ils/issues/475>`__: redirects
+      homepage to the global view.
+   -  `#540 <https://github.com/rero/rero-ils/issues/540>`__: helps
+      distinguish organisation from organisation as an author in order
```
Comment (mmo) [inline RELEASE-NOTES.rst]: distinguish "organisation" as a RERO ILS client from "organisation" as an author, in order

File: RELEASE-NOTES.rst [hunk 3]
```diff
@@ -18,6 +18,55 @@
...
+      to ease translation.
+   -  `#573 <https://github.com/rero/rero-ils/issues/573>`__: rename
+      online locations that aren’t not online location.
+   -  `#232 <https://github.com/rero/rero-ils/issues/232>`__: improves
+      position of flash messages as to always get the message, even if
+      the screen is scrolled down.
```
Comment (mmo) [inline RELEASE-NOTES.rst]: position of flash messages in order to always show on the screen, even if it is scrolled down.

File: RELEASE-NOTES.rst [hunk 4]
```diff
@@ -18,6 +18,55 @@
...
+   -  `#556 <https://github.com/rero/rero-ils/issues/556>`__: fixes a
+      wrong labelled field in the ``identifiedBy`` field.
```
Comment (mmo) [inline RELEASE-NOTES.rst]: wrong label in the `identifiedBy` field.

File: RELEASE-NOTES.rst [hunk 5]
```diff
@@ -18,6 +18,55 @@
...
+      bug in the location editor which was preventing the submit button
+      to be activated.
+   -  `#404 <https://github.com/rero/rero-ils/issues/404>`__: fixes
+      internationalization code in the source in order to make it
+      translatable.
+   -  `#553 <https://github.com/rero/rero-ils/issues/553>`__: fixes
+      wrong filter on persons detailed views to restore the list of
+      related documents.
+   -  `#560 <https://github.com/rero/rero-ils/issues/560>`__: makes sure
+      fees are indexed in ElastiSearch.
```
Comment (mmo) [inline RELEASE-NOTES.rst]: fees are indexed in Elasticsearch. (missing 'c')

File: RELEASE-NOTES.rst [hunk 6]
```diff
@@ -18,6 +18,55 @@
...
+
+   -  `#488 <https://github.com/rero/rero-ils/issues/488>`__: disallows
+      editing libraries by a non affiliated librarian.
+   -  `#475 <https://github.com/rero/rero-ils/issues/475>`__: redirects
+      homepage to the global view.
+   -  `#540 <https://github.com/rero/rero-ils/issues/540>`__: helps
+      distinguish organisation from organisation as an author in order
+      to ease translation.
+   -  `#573 <https://github.com/rero/rero-ils/issues/573>`__: rename
+      online locations that aren’t not online location.
```
Comment (mmo) [inline RELEASE-NOTES.rst]: locations incorrectly labelled as online.

**Review by pronguen — APPROVED** [review]

**Pull request merged**